DoseMathTests/DoseMathTests.swift

```diff
@@ -1504,3 +1504,232 @@ class RecommendBolusTests: XCTestCase {
         XCTAssertEqual(1.96, dose.amount, accuracy: 1.0 / 40.0)
     }
 }
+
+class automaticIOBLimitTests: XCTestCase {
```

Comment:
Class names are always capitalized.  But I don't know that this should be a separate class. I see that we do have two classes in this file, with some duplication of methods, but I think these tests should fit into those existing classes.

Reply:
Moved all new tests into class RecommendTempBasalTests